### ISSUE TITLE
Dart analysis server error: Element not found in summary

### ISSUE THREAD

**Issue by rayk** [dart-lang/sdk#34555]
Analyzer Feedback from IntelliJ

# Version information

- `IDEA 183.2635.17`
- `2.1.0-dev.5.0`
- `WS-183.2635.17, JRE 1.8.0_152-release-1293-b10x64 JetBrains s.r.o, OS Mac OS X(x86_64) v10.13.6, screens 2560x1440, 1200x1920`

# Exception

Please append the contents of:
file:///private/var/folders/k3/rpf30xp576gd3b7wdpvvyg2r0000gn/T/report.txt

```
Dart analysis server, SDK version 2.1.0-dev.5.0, server version 1.20.5, error: Analysis failed: /Users/rayk/Projects/app/lib/src/services/state/root.dart
Exception: Element not found in summary: package:app/src/services/state/root.dart

**Comment by empirefox** [comment]
I also got that error from Android Studio today, but build always success without any error. Everything is ok now after i do:

```
  -  final Completer _sinkDone = Completer();
  + final Completer<bool> _sinkDone = Completer();
```

**Comment by stereotype441** [comment]
Hopefully fixed by e648d63bedc2a4decb3b8cbcf0f2e1c44db960b0.  Unfortunately since we don't have a reliable repro we will have to wait and see.  I'll reopen if a similar problem is seen in a version including this commit.
